Blockzilla/SettingsViewController.swift

```diff
@@ -357,23 +365,62 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
+            let attributedSubtitle = NSMutableAttributedString(string: subtitle)
+            attributedSubtitle.append(space)
+            attributedSubtitle.append(learnMore)
+            cell.detailTextLabel?.attributedText = attributedSubtitle
+            let tapGesture = UITapGestureRecognizer(target: self, action: selector)
+            cell.addGestureRecognizer(tapGesture)
+        }
+        return cell;
+    }
+    
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         var cell: UITableViewCell
         switch sections[indexPath.section] {
+        case .privacy:
+            if indexPath.row == 0 {
+                cell = SettingsTableViewCell(style: .subtitle, reuseIdentifier: "trackingCell")
+                cell.textLabel?.text = String(format: UIConstants.strings.trackingProtectionLabel)
+                cell.accessibilityIdentifier = "settingsViewController.trackingCell"
+                cell.accessoryType = .disclosureIndicator
+            } else {
+                cell = getToggleCell(indexPath: indexPath)
+            }
         case .search:
-            guard let searchCell = tableView.dequeueReusableCell(withIdentifier: "accessoryCell") as? SettingsTableViewAccessoryCell else { fatalError("Accessory cells do not exist") }
-
-            let label = indexPath.row == 0 ? UIConstants.strings.settingsSearchLabel : UIConstants.strings.settingsAutocompleteSection
-            let autocompleteLabel = Settings.getToggle(.enableDomainAutocomplete) || Settings.getToggle(.enableCustomDomainAutocomplete) ? UIConstants.strings.autocompleteCustomEnabled : UIConstants.strings.autocompleteCustomDisabled
-            let accessoryLabel = indexPath.row == 0 ? searchEngineManager.activeEngine.name : autocompleteLabel
-            let identifier = indexPath.row == 0 ? "SettingsViewController.searchCell" : "SettingsViewController.autocompleteCell"
-
-            searchCell.accessoryLabelText = accessoryLabel
-            searchCell.labelText = label
-            searchCell.accessibilityIdentifier = identifier
-
-            cell = searchCell
+            if(indexPath.row < 2){
+                guard let searchCell = tableView.dequeueReusableCell(withIdentifier: "accessoryCell") as? SettingsTableViewAccessoryCell else { fatalError("Accessory cells do not exist") }
+                let autocompleteLabel = Settings.getToggle(.enableDomainAutocomplete) || Settings.getToggle(.enableCustomDomainAutocomplete) ? UIConstants.strings.autocompleteCustomEnabled : UIConstants.strings.autocompleteCustomDisabled
+                let labels : (label : String, accessoryLabel : String, identifier : String) = indexPath.row == 0 ? (UIConstants.strings.settingsSearchLabel,searchEngineManager.activeEngine.name, "SettingsViewController.searchCell") : (UIConstants.strings.settingsAutocompleteSection,autocompleteLabel,"SettingsViewController.autocompleteCell")
```

Comment:
Nit but this line is extremely long, probably just enter it to make it more human friendly.